xsuite/src/world/fsworld.test.ts

```diff
@@ -436,6 +436,44 @@ test.concurrent("FSWorld.transfer", async () => {
   });
 });
 
+test.todo.concurrent(
```

Comment:
Try to implement those tests with gasPrice = 0 or gasLimit = 0 and ideally use the same txs for LS and FS